integration-cli/docker_cli_daemon_test.go

```diff
@@ -2070,3 +2070,20 @@ func (s *DockerDaemonSuite) TestRunLinksChanged(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf(out))
 	c.Assert(out, check.Not(checker.Contains), "1 packets transmitted, 1 packets received")
 }
+
+func (s *DockerDaemonSuite) TestDaemonUngracefulShutdownCleanUpRootfsOfOverlay(c *check.C) {
```

Comment:
Oh, I just think the CI environment is using `overlay`. yes, it require `/` to be mounted as overlay. why check if the container was removed successful is related the issue #19679. I'm going to remove this test case since `TestCleanupMountsAfterCrash` is enough.